dist/bin/dotr

```diff
@@ -58,16 +58,16 @@ done
 
 if [ $run_repl == true ] || [ ${#residual_args[@]} -eq 0 ]; then
   if [ "$CLASS_PATH" ]; then
-    cp_arg="-classpath $CLASS_PATH"
+    cp_arg="$CLASS_PATH"
   fi
   echo "Starting dotty REPL..."
-  eval "$PROG_HOME/bin/dotc $cp_arg -repl ${residual_args[@]}"
+  eval "$PROG_HOME/bin/dotc -classpath \"$cp_arg\" -repl ${residual_args[@]}"
```

Comment:
The semantic is different here. Now if you pass no classpath, you will run `dotc -classpath "" ...`,
before `dotc  ...`. `dotc` is not happy if you pass an empty classpath

Reply:
I've updated the PR, and added a test to packtest to guard against regression for empty classpath: https://github.com/lampepfl/packtest/pull/9/files